proposed/2021/NuGetConsumePDBInPackageReference.md

```diff
@@ -0,0 +1,216 @@
+# Consuming pdb support with PackageReference
+
+* Status: Reviewing
+* Author(s): [Heng Liu](https://github.com/heng-liu)
+* Issue: [5926](https://github.com/NuGet/Home/issues/5926) Consume pdbs from packages in PackageReference - add Debug Symbols to LockFileTargetLibrary
+* Type: Feature
+
+## Problem Background
+
+In the PackageReference world, the packages are consumed from the global-packages folder, .pdb, .xml files from lib and runtime folder are not copied into the build output folder.
+In some cases, consuming .pdb, .xml files are needed to support certain features, like debugging ( see issue [1458](https://github.com/dotnet/sdk/issues/1458)).
+The established design for consuming files out of packages is that NuGet tells the .NET SDK which files are important through the assets file, or to be specific, the [LockFileTargetLibrary](https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.ProjectModel/LockFile/LockFileTargetLibrary.cs). 
+Currently, NuGet does not select pdb or xml files.
+
+In comparison, in packages.config, .pdb, .xml files are automatically copied from lib folder into the build output folder because of the [ResolveAssemblyReference](https://github.com/dotnet/msbuild/blob/main/src/Tasks/AssemblyDependency/ResolveAssemblyReference.cs#L75) task. So there is no such problem in packages.config.
+
+## Who are the customers
+
+Partner team(.NET SDK) who needs to consume .pdb, .xml files for their features in PackageReference world. 
+
+Take debugging as an example. When a NuGet package includes a .pdb file in its lib folder, in order to debug into the NuGet package, it needs the .pdb file to be copied to the build output folder of the project which references the NuGet package.
+
+## Requirements
+
+* A feature to allow the .NET SDK to consume .pdb, .xml files.
+
+## Goals
+
+* Define an approach to make .NET SDK be able to consume .pdb, .xml files.
+
+## Non-Goals
+
+* .NET SDK side change to consume .pdb or .xml.
```

Comment:
Do you mean that it is a non-goal of _this document_ to define how the SDK needs to change to consume the new information? Or that when given an updated NuGet the SDK should Just Work without changes?

Reply:
Sorry for the confusion caused. It means this document won't define how the SDK needs to change to consume the related property.